experiments/compositions/composition/Makefile

```diff
@@ -1,10 +1,15 @@
 # GCP project to use for pushing manifests
 GCP_PROJECT_ID ?= $(shell gcloud config get-value project)
 # Image URL to use all building/pushing image targets
-IMG ?= gcr.io/$(GCP_PROJECT_ID)/composition:latest
-INLINE_IMG ?= gcr.io/$(GCP_PROJECT_ID)/manifests-inline:latest
+GIT_IMG_VERSION ?= $(shell git rev-parse --short HEAD)
+IMG_VERSION ?= v0.0.1.alpha
+IMG ?= gcr.io/$(GCP_PROJECT_ID)/composition:$(IMG_VERSION)
+INLINE_IMG ?= gcr.io/$(GCP_PROJECT_ID)/manifests-inline:$(IMG_VERSION)
```

Comment:
Do we want a common prefix for the images?

Reply:
added.